apps/inline-help/webpack.config.js

```diff
@@ -90,6 +90,7 @@ module.exports = {
 		new webpack.IgnorePlugin( { resourceRegExp: /^\.\/locale$/, contextRegExp: /moment$/ } ),
 		new ExtensiveLodashReplacementPlugin(),
 		new InlineConstantExportsPlugin( /\/client\/state\/action-types.js$/ ),
+		new webpack.NormalModuleReplacementPlugin( /^path$/, 'path-browserify' ),
```

Comment:
Copied from the main client webpack configuration. It resolves the following error which occurs after building the app after running `yarn workspaces focus`:

```sh
[build:inline-help] [webpack-cli] ModuleNotFoundError: Module not found: Error: Can't resolve 'path' in '/Users/noahtallen/source/wp-calypso/client/lib/media/utils'
[build:inline-help] 
[build:inline-help] BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
[build:inline-help] This is no longer the case. Verify if you need this module and configure a polyfill for it.
[build:inline-help] 
[build:inline-help] If you want to include a polyfill, you need to:
[build:inline-help]     - add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
[build:inline-help]     - install 'path-browserify'
[build:inline-help] If you don't want to include a polyfill, you can use an empty module like this:
[build:inline-help]     resolve.fallback: { "path": false }
[build:inline-help]     at /Users/noahtallen/source/wp-calypso/node_modules/webpack/lib/Compilation.js:2011:28
[build:inline-help]     at /Users/noahtallen/source/wp-calypso/node_modules/webpack/lib/NormalModuleFactory.js:795:13
[build:inline-help]     at eval (eval at create (/Users/noahtallen/source/wp-calypso/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
[build:inline-help]     at /Users/noahtallen/source/wp-calypso/node_modules/webpack/lib/NormalModuleFactory.js:275:22
[build:inline-help]     at eval (eval at create (/Users/noahtallen/source/wp-calypso/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
[build:inline-help]     at /Users/noahtallen/source/wp-calypso/node_modules/webpack/lib/NormalModuleFactory.js:431:22
[build:inline-help]     at /Users/noahtallen/source/wp-calypso/node_modules/webpack/lib/NormalModuleFactory.js:124:11
[build:inline-help]     at /Users/noahtallen/source/wp-calypso/node_modules/webpack/lib/NormalModuleFactory.js:667:25
[build:inline-help]     at /Users/noahtallen/source/wp-calypso/node_modules/webpack/lib/NormalModuleFactory.js:852:8
[build:inline-help]     at /Users/noahtallen/source/wp-calypso/node_modules/webpack/lib/NormalModuleFactory.js:972:5
```